data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,110 @@
-
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "pattern": "^https:\\/\\/hdl\\.handle\\.net\\/20\\.5000\\.1025\\/.{3}-.{3}-.{3}$",
+      "examples": ["https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"],
+      "description": "PID of the object."
+    },
+   "rdf:type": {
+      "type": "string",
+      "const": "VirtualCollections",
```

Comment:
Should this be "VirtualCollections" or should it point to the FDO type record as we do with other types?

Reply:
i think for our data model schema, we have been using string so let's keep this string now. but we need to align it for all the schema if we point to the FDO type. I think we do it in the FDO Profile schema.